api/web3server.go

```diff
@@ -244,6 +248,8 @@ func (svr *Web3Server) handlePOSTReq(req *http.Request) interface{} {
 			}
 		case "eth_newBlockFilter":
 			res, err = svr.newBlockFilter()
+		case "eth_debugTransactionByHash":
+			res, err = svr.debugTransactionByHash(params)
```

Comment:
we add this new func, so babel can call it for debugging purpose?

Reply:
The API was added expressly for this.  https://ropsten.etherscan.io/vmtrace?txhash=0x0e55ee6c5d9a1e68f628186ce172f883d95e2b2aa61a77f65485aa6fef890467